src/components/contactUsPage/SideBar.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { MDBCard } from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+function ContactInformation() {
+  const { t } = useTranslation();
+  const contact = t('contactUsPage.contactUs');
+  return (
+    <>
+      <div className="flex justify-end">
+        <MDBCard className="max-h-full bg-blue-800 w-1/4 flex justify-center">
+          <h1 className=" my-4 pb-2 text-center white-text text-2xl">
+            {contact}
```

Comment:
I don't know but I think using the translation directly is better
```suggestion
            {t('contactUsPage.contactUs')}
```

Reply:
Why do you think that?
using translation directly mean you will call the function more than one time and that is a bit slow I believe
I prefer this method

Reply:
The json file is already loaded once the app start during the load of app.js and she is only using it once and it is just a typing thing it will make it harder for a reader to read it as a variable on the top part of the function